packages/wrangler/package.json

```diff
@@ -73,7 +73,8 @@
 		"miniflare": "workspace:*",
 		"path-to-regexp": "6.3.0",
 		"unenv": "2.0.0-rc.19",
-		"workerd": "1.20250726.0"
+		"workerd": "1.20250726.0",
+		"zod": "^4.0.13"
```

Comment:
comment: zod is used in miniflare so maybe we should add & use a catalog version - miniflare uses 3.22.3